manifests/init.pp

```diff
@@ -160,6 +164,22 @@
     notify    => Exec['root newaliases'],
   }
 
+  file {'/etc/profile.d/root_logout_timeout.sh':
```

Comment:
For these files if the default is `undef` you could do something like this:

```
if $logout_timeout {
  $timeout_ensure = 'file'
} else {
  $timeout_ensure = 'absent'
}

file {'/etc/profile.d/root_logout_timeout.sh':
  ensure => $timeout_ensure,
  ...
}
```

The rational behind this is I personally don't want to deploy these files in my environment and so it would be good for those files to be opt-in if a timeout is set.

Reply:
I'll whip this up shortly.